packages/product-json-to-csv/test/writer.spec.js

```diff
@@ -163,17 +163,51 @@ describe('Writer', () => {
       const output = tempFile.name
       const outputStream = fs.createWriteStream(output)
 
+      let entries = 0
       // Extract data from zip file and test
       outputStream.on('finish', () => {
         fs.createReadStream(output)
           .pipe(unzip.Parse())
           .on('entry', entry => {
+            entries += 1
             if (entry.path === 'products/product-type-1.csv')
               entry.pipe(verifyCsv1)
             else if (entry.path === 'products/product-type-2.csv')
               entry.pipe(verifyCsv2)
+
+            if (entries === 2) {
+              expect(entries).toEqual(2)
+              tempFile.removeCallback()
+              done()
```

Comment:
This is painful to the eyes, we should make time asap to make a better implementation.

Reply:
Agree .. I will investigate how does it work in that unzip module and try to fix it there